dataset.proto

```diff
@@ -34,9 +44,11 @@ message DataSet {
     string url = 1;
   }
 
-  Categorization categorization = 1;
-  AdFraud        adfraud        = 2;
-  // 3 and 4 are reserved
-  Malicious      malicious      = 5;
-  Echo           echo           = 6;
+  Categorization         categorization = 1;
+  AdFraud                adfraud        = 2;
+  // 3 is reserved
+  Keyword                keyword        = 4;
```

Comment:
I don't think it needs to be enforced in this repo, because if a customer does request both (mistakenly) then at response time we can copy the keys in sentiment to the array for keywords and give it to them...for a price.